src/spox/opset/ai/onnx/ml/v3.py

```diff
@@ -662,13 +660,22 @@ def array_feature_extractor(
     Type constraints:
      - T: `tensor(double)`, `tensor(float)`, `tensor(int32)`, `tensor(int64)`, `tensor(string)`
     """
-    return _ArrayFeatureExtractor(
-        _ArrayFeatureExtractor.Attributes(),
-        _ArrayFeatureExtractor.Inputs(
-            X=X,
-            Y=Y,
-        ),
-    ).outputs.Z
+    input_prop_values: PropDict = {
+        "X": get_value(X),
+        "Y": get_value(Y),
+    }
+    return (
+        _ArrayFeatureExtractor(
+            _ArrayFeatureExtractor.Attributes(),
+            _ArrayFeatureExtractor.Inputs(
+                X=unwrap_vars(X),
+                Y=unwrap_vars(Y),
+            ),
+            input_prop_values=input_prop_values,
+        )
+        .get_output_vars(input_prop_values=input_prop_values)
```

Comment:
It is a bit surprising that we would need to supply `input_prop_values` twice here. Could we make it so that we only need to provide them to `get_output_vars`?

Reply:
done